src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -130,7 +131,13 @@ public function getLogDir(): string
 
     public function registerBundles(): iterable
     {
-        $contents = require $this->getBundlesPath();
+        if (!is_file($bundlesPath = $this->getBundlesPath())) {
+            yield new FrameworkBundle();
```

Comment:
shouldn't this default bundle list also be mentioned in the changelog?

Reply:
Indeed, I missed that, thanks!